Utilities/bootstrap

```diff
@@ -992,7 +1003,7 @@ def main():
     # We need to embed an RPATH so swift-{build,test} can find the core
     # libraries.
     if platform.system() == 'Linux':
-        embed_rpath = "$ORIGIN/../lib/swift/linux"
+        embed_rpath = "$ORIGIN/../" + args.libdir + "/swift/linux"
```

Comment:
@aciidb0mb3r not a Python export, but wouldn't using `os.path.join` be better?

Reply:
probably not in this case since its something we want to embed as-is.